app/containers/TodoApp/components/TodoList.js

```diff
@@ -0,0 +1,63 @@
+/* @flow */
+
+import React from 'react';
+import Relay from 'react-relay';
+import TodoItem from './TodoItem';
+
+type TodoEdge = {
+  node: {
+    id: string,
+  },
+};
+
+type Props = {
+  todos: {
+    edges: Array<TodoEdge>,
+  },
+  user: Object
+};
+
+/**
+ * Todo list component
+ * Parent component: TodoApp
+ * Child component: TodoItem
+ */
+class TodoList extends React.Component {
```

Comment:
can this just be a functional component (`const TodoList = props => (<div />)`) or does the relay container need a component extended from React.Component?